<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QStarDict::SettingsDialog</class>
 <widget class="QDialog" name="QStarDict::SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QStarDict Settings</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGlobalSettings">
      <attribute name="title">
       <string>Global settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Application</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="runInBackgroundBox">
            <property name="text">
             <string>Run in background</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="autostartBox">
            <property name="text">
             <string>Start with system</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Translations browser</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="instantSearchBox">
            <property name="text">
             <string>Instant search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="linksBox">
            <property name="text">
             <string>Show words that are translatable as links</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="linksModifierContainer" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="linksModifierBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Only if modifier is pressed</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="linksModifierKeyBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>Alt</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Control</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Shift</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Win</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="speechCmdLabel">
              <property name="text">
               <string>Pronounce words using this command:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="speechCmdEdit">
              <property name="toolTip">
               <string>Enter cmd for the speaching program.&lt;br&gt;If cmd contains &quot;%s&quot; it will be replaced to word, else word will be written to stdin of speech process.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>426</width>
           <height>318</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDictionaries">
      <attribute name="title">
       <string>Dictionaries</string>
      </attribute>
      <layout class="QVBoxLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Dictionaries</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QTableView" name="dictsTableView">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <widget class="QToolButton" name="dictsMoveUpButton">
              <property name="toolTip">
               <string>Move up</string>
              </property>
              <property name="text">
               <string>Up</string>
              </property>
              <property name="icon">
               <iconset resource="pixmaps/pixmaps.qrc">
                <normaloff>:/icons/arrow-up.png</normaloff>:/icons/arrow-up.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="dictsMoveDownButton">
              <property name="toolTip">
               <string>Move down</string>
              </property>
              <property name="text">
               <string>Down</string>
              </property>
              <property name="icon">
               <iconset resource="pixmaps/pixmaps.qrc">
                <normaloff>:/icons/arrow-down.png</normaloff>:/icons/arrow-down.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="dictsShowInfoButton">
              <property name="toolTip">
               <string>Show information about dictionary</string>
              </property>
              <property name="text">
               <string>Show info</string>
              </property>
              <property name="icon">
               <iconset resource="pixmaps/pixmaps.qrc">
                <normaloff>:/icons/dialog-information.png</normaloff>:/icons/dialog-information.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>196</height>
          </size>
         </property>
         <property name="title">
          <string>Plugins</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QStarDict::PluginsView" name="pluginsTableView">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabPopupWindow">
      <attribute name="title">
       <string>Popup window</string>
      </attribute>
      <layout class="QGridLayout">
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Behavior</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QCheckBox" name="useScanBox">
            <property name="text">
             <string>Scan selection</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="showIfNotFoundBox">
            <property name="text">
             <string>Show if word not found</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="shortcutPopupBox">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Show translation popup by shortcut</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="shortcutPopupEdit">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="scanModifierContainer" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="useScanModifierBox">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="text">
                <string>Show only if modifier is pressed</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="tristate">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="modifierKeyBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Timeout before hide after mouse over</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="timeoutBeforeHideSpin">
              <property name="specialValueText">
               <string>Don't hide</string>
              </property>
              <property name="suffix">
               <string> sec</string>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="minimum">
               <double>0.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.500000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="pronounceWordBox">
              <property name="text">
               <string>Pronounce the word</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Appearance</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="2" column="0">
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Opacity</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="popupOpacitySpin">
              <property name="suffix">
               <string>%</string>
              </property>
              <property name="prefix">
               <string/>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Default width</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="popupDefaultWidthSpin">
              <property name="minimum">
               <number>96</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>320</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Default height</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="popupDefaultHeightSpin">
              <property name="minimum">
               <number>96</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
              <property name="value">
               <number>240</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>101</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Appearance</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QStarDict::CSSEdit" name="appearanceCSSEdit" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMisc">
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Other Plugins</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QStarDict::PluginsView" name="miscPluginsView">
            <property name="gridStyle">
             <enum>Qt::NoPen</enum>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QStarDict::CSSEdit</class>
   <extends>QWidget</extends>
   <header>cssedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QStarDict::PluginsView</class>
   <extends>QTableView</extends>
   <header>pluginsview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>instantSearchBox</tabstop>
  <tabstop>speechCmdEdit</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>dictsTableView</tabstop>
  <tabstop>dictsMoveUpButton</tabstop>
  <tabstop>dictsMoveDownButton</tabstop>
  <tabstop>dictsShowInfoButton</tabstop>
  <tabstop>pluginsTableView</tabstop>
  <tabstop>useScanBox</tabstop>
  <tabstop>showIfNotFoundBox</tabstop>
  <tabstop>useScanModifierBox</tabstop>
  <tabstop>modifierKeyBox</tabstop>
  <tabstop>timeoutBeforeHideSpin</tabstop>
  <tabstop>pronounceWordBox</tabstop>
  <tabstop>popupDefaultWidthSpin</tabstop>
  <tabstop>popupDefaultHeightSpin</tabstop>
  <tabstop>popupOpacitySpin</tabstop>
 </tabstops>
 <resources>
  <include location="pixmaps/pixmaps.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QStarDict::SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QStarDict::SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useScanModifierBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>modifierKeyBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>shortcutPopupBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>shortcutPopupEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>linksModifierBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>linksModifierKeyBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useScanBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>scanModifierContainer</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>linksBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>linksModifierContainer</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
